Sources/PT.PM/LanguageDetectors/ParserLanguageDetector.cs

```diff
@@ -48,6 +51,17 @@ public override DetectionResult Detect(TextFile sourceFile, IEnumerable<Language
                 return new DetectionResult(langs[0]);
             }
 
+            if (langs.Any(l => l.IsSql()))
```

Comment:
Now I think it's more appropriate to use `All` instead of `Any`.

Reply:
No, cause in some cases, when languages argument is null, we have list with all languages, and in this case if any is SQL, we need to use Any() func